services/madoc-ts/src/frontend/admin/pages/content/manifests/manifest-projects.tsx

```diff
@@ -40,14 +41,13 @@ export const ManifestProjects = createUniversalComponent<ManifestProjectsType>(
         <p>This manifest is in the following projects.</p>
         <hr />
         {projects.map(project => (
-          <div key={project.id}>
-            <Link to={`/projects/${project.id}`}>
-              <LocaleString as="h3">{project.label}</LocaleString>
-            </Link>
-            <p>
-              <LocaleString>{project.summary}</LocaleString>
-            </p>
-          </div>
+          <SnippetLarge
+            label={<LocaleString>{project.label}</LocaleString>}
+            portrait={true}
+            subtitle={<LocaleString>{project.summary}</LocaleString>}
+            link={`./../../projects/${project.id}`}
```

Comment:
I'm not happy with what I've had to do here. The previous `<Link to=...>` approach generated the `href` relative to the current site (i.e. `localhost:8888/s/default/madoc/admin`) whereas using the same expression here generated it relative to the root URL (i.e. `localhost:8888`). I really don't like the fact that I've had to give this component knowledge of how it's used in the routes

Reply:
I think you should be able to use
```
link={`/projects/${project.id}`}
```

That will take you to the admin page for the projects. The absolute path are all under `/s/default/madoc/admin` for these components and then everything under `site` is `/s/default/madoc`. Unfortunately you can't route between them without a page-refresh as they are completely separate React apps.

Going to look at cross-linking more generally so you can jump from a project or collection page on the admin to the site and back again.

Reply:
Unfortunately that doesn't work - the `a` tag ends up as:

```
<a role="link" href="/projects/1" class="sc-fzppAM fqNlnf">view project</a>
```

Reply:
ahh, it's creating an anchor tag directly. I think if you add a prop: `linkAs={HrefLink}` (in the shared utilities) that will fix the issue 👍